apps/etterlatte-behandling/src/main/kotlin/sak/SakTilgangDao.kt

```diff
@@ -83,4 +83,27 @@ class SakTilgangDao(private val datasource: DataSource) {
             }
         }
     }
+
+    fun hentSakMedGraderingOgSkjermingPaaKlage(klageId: String): SakMedGraderingOgSkjermet? {
```

Comment:
Jeg vil gjerne ha en test eller to på denne :)

Reply:
Tenker å få opp noe testing av hele greia også, bare er litt 🤠 i starten